internal/integration/models/models.go

```diff
@@ -41,6 +41,8 @@ type DataUpPayload struct {
 	RXInfo          []RXInfo          `json:"rxInfo,omitempty"`
 	TXInfo          TXInfo            `json:"txInfo"`
 	ADR             bool              `json:"adr"`
+    SF              int               `json:"sf"`
```

Comment:
Please note that this struct exists for backwards compatibility, when `marshaler` is configured to `json_v3`. It is not used by the InfluxDB integration.